libsplinter/src/rest_api/mod.rs

```diff
@@ -564,6 +569,8 @@ pub struct RestApiBuilder {
     bind: Option<String>,
     #[cfg(feature = "rest-api-cors")]
     whitelist: Option<Vec<String>>,
+    #[cfg(all(feature = "rest-api-auth", feature = "oauth"))]
+    oauth_provider: Option<OAuthProvider>,
```

Comment:
This isn't configuration code, as it contains the oauth Client instance.  The forthcoming endpoint to provide the CLI with correct information will need the data that was provided to this OAuthProvider. Its current implementation cannot return the values, such as token url or client id/secret that the CLI will need.

Reply:
Ah, right, forgot about that endpoint. Will update.

Reply:
After conversations with @peterschwarz, we agreed to leave this as-is because we will likely not need to return the values.